<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SCi</string>
  </property>
  <property name="windowIcon">
   <iconset resource="photos.qrc">
    <normaloff>:/Photos/icons/512.ico</normaloff>:/Photos/icons/512.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="ButtonToday">
           <property name="text">
            <string>今日</string>
           </property>
           <property name="icon">
            <iconset resource="photos.qrc">
             <normaloff>:/icon_button/Photos/icons/today_button_icon.png</normaloff>:/icon_button/Photos/icons/today_button_icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonClassSchedule">
           <property name="text">
            <string>课程表</string>
           </property>
           <property name="icon">
            <iconset resource="photos.qrc">
             <normaloff>:/icon_button/Photos/icons/schedule_button_icon.png</normaloff>:/icon_button/Photos/icons/schedule_button_icon.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonMap">
           <property name="text">
            <string>地图</string>
           </property>
           <property name="icon">
            <iconset resource="photos.qrc">
             <normaloff>:/icon_button/Photos/icons/map_button_icon.png</normaloff>:/icon_button/Photos/icons/map_button_icon.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonNotice">
           <property name="text">
            <string>公告</string>
           </property>
           <property name="icon">
            <iconset resource="photos.qrc">
             <normaloff>:/icon_button/Photos/icons/notice_button_icon.png</normaloff>:/icon_button/Photos/icons/notice_button_icon.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="FindButton">
           <property name="text">
            <string>查找</string>
           </property>
           <property name="icon">
            <iconset resource="photos.qrc">
             <normaloff>:/icon_button/Photos/icons/find_button_icon.png</normaloff>:/icon_button/Photos/icons/find_button_icon.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SettingButton">
           <property name="text">
            <string>设置</string>
           </property>
           <property name="icon">
            <iconset resource="photos.qrc">
             <normaloff>:/icon_button/Photos/icons/settings_button_icon.png</normaloff>:/icon_button/Photos/icons/settings_button_icon.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>462</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources>
  <include location="photos.qrc"/>
 </resources>
 <connections/>
</ui>
